greynoise/plugin.spec.yaml

```
@@ -0,0 +1,509 @@
+---
+
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: greynoise
+title: GreyNoise
+vendor: greynoise
+support: community
+status: []
+description: Interact with GreyNoise API
+version: 1.0.0
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/greynoise
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://greynoise.io
+tags:
+  - greynoise
+  - ip
+  - threat intel
+  - intelligence
+hub_tags:
+  use_cases: [data_enrichment, threat_detection_and_response]
+  keywords: [ip, intelligence, greynoise]
+  features: []
+enable_cache: true
```

If the cache is not used we can set this to False

```suggestion
enable_cache: false
```